src/main/java/org/ow2/proactive/catalog/service/CatalogObjectService.java

```diff
@@ -349,26 +360,45 @@ public CatalogObjectMetadata updateObjectMetadata(String bucketName, String name
             throw new TagsIsNotValidException(tags.get(), "tags");
         }
 
-        String previousProjectName = catalogObjectRevisionEntity.getProjectName();
-        String previousTags = catalogObjectRevisionEntity.getTags();
-        if (!catalogObjectRevisionEntity.getProjectName().equals(projectName.orElse("")) ||
-            !catalogObjectRevisionEntity.getTags().equals(tags.orElse(""))) {
-            buildCatalogObjectRevisionEntity(UPDATE_COMMIT_MESSAGE,
-                                             username,
-                                             projectName.orElse(previousProjectName),
-                                             tags.orElse(previousTags),
-                                             catalogObjectRevisionEntity.getRawObject(),
-                                             catalogObjectEntity,
-                                             KeyValueLabelMetadataHelper.convertFromEntity(catalogObjectRevisionEntity.getKeyValueMetadataList()),
-                                             true);
-        }
-        kind.ifPresent(catalogObjectEntity::setKind);
-        kind.ifPresent(catalogObjectEntity::setKindLower);
-        catalogObjectEntity.setNameLower(name);
-        contentType.ifPresent(catalogObjectEntity::setContentType);
-        contentType.ifPresent(catalogObjectEntity::setContentTypeLower);
-        catalogObjectRepository.save(catalogObjectEntity);
-        return new CatalogObjectMetadata(catalogObjectEntity);
+        List<CatalogObjectRevisionEntity> updateRevisionEntities = new ArrayList<>();
+        List<CatalogObjectEntity> updateCatalogObjectEntities = new ArrayList<>();
+
+        bucketNameObjectName.keySet().forEach(bucketName -> {
+            findBucketByNameAndCheck(bucketName);// TODO je pense pas necessaire car en dessous ca check aussi
```

Comment:
if there is a TODO, it should be in english

Reply:
I actually forgot to remove this. Thanks